bulbs/poll/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+class Poll(models.Model):
```

Comment:
Is the Poll not going to be part of the general Content schema? Should we be able to search for it in the index?